model/graph.py

```diff
@@ -329,21 +336,20 @@ def __init__(self, graph, input_sizes, input_dims,
             per_input_projections.append(integrator_conv)
             
             self.topdown_projections.append(nn.ModuleList(per_input_projections))
-            #self.topdown_projections = nn.ModuleList(self.topdown_projections)
             
-        #print(self.topdown_projections)
 
     def forward(self, all_inputs, batch=True):
         """
-        :param a list of tensor of size n, each consisting a input_tensor: (b, t, c, h, w). 
-            n as the number of input signals. Order should correspond to self.graph.input_node_indices
-        :param topdown: size (b,hidden,h,w) 
-        :return: label 
+        :param all_inputs 
+               list of tensor of size n, each consisting a input_tensor: (b, t, c, h, w). 
+               n as the number of input streams. Order should correspond to self.graph.input_node_indices
+        :return: predicted label 
         """
-        #find the time length of the longest input signal + enough extra time for the last input to go through all nodes
+        # find the time length of the longest input signal + enough extra time for the last input to go through all nodes
         seq_len = max([i[0].shape[1] if isinstance(i, list) else i.shape[1] for i in all_inputs])
-        process_time = seq_len + self.graph.output_node_index - 1
+        process_time = seq_len + self.graph.output_node_index
```

Comment:
This line is the legacy version. Will push the updated version with longest_path_length